README.md

```diff
@@ -1,8 +1,35 @@
-# Kurtosis Usage Examples
+# Awesome Kurtosis [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
 
-This is the official Kurtosis examples library.
+<img src="./logo.png" width="500">
 
-Each example resides in a separate folder. We currently offer the following different types of projects:
+> A curated list of Kurtosis Starlark Packages
 
-* Writing a unit test using the Kurtosis Golang SDK
-* Running a Kurtosis Starlark package
+These Kurtosis packages and examples provide a starting point for how to setup and test different environments using a Kurtosis Stalrlark package and
+to manage their lifetime using the Kurtosis CLI.
+
+> **Note**
+> The following samples are intended for use in local development and testing environments. These samples must not be deployed in production environments.
+
+## Contents
+
+- [Kurtosis Starlark Packages in the Web2 Space](#kurtosis-starlark-web2-examples)
+- [Kurtosis Starlark Packages in the Web3 Space](#kurtosis-starlark-web3-examples)
+- [Tests and other examples](#tests-and-other-examples)
+
+## Kurtosis Starlark Web2 Examples
+
+- [Cassandra Package](https://github.com/kurtosis-tech/cassandra-package) - A Kurtosis Starlark Package that spins up an n node Cassandra cluster and verifies that the cluster that was spun up was healthy
+- [Datastore Army Package](https://github.com/kurtosis-tech/datastore-army-package) - A Kurtosis Starlark Package that spins up an n node Datastore server cluster
+
+## Kurtosis Starlark Web3 Examples
+
+- [eth2 Package](https://github.com/kurtosis-tech/eth2-package) - A Kurtosis Starlark Package that spins up a local Ethereum testnet, supporting supporting 9 different EL and CL clients including geth, lighthouse, lodestar, nimbus and erigon.
+- [near Package](https://github.com/kurtosis-tech/near-package) - A Kurtosis Starlark package that spins up a local near testnet with an indexer, wallet, explorer and more.
```

Comment:
```suggestion
- [NEAR Package](https://github.com/kurtosis-tech/near-package) - A Kurtosis Starlark package that spins up a local NEAR testnet with a local RPC endpoint, a NEAR explorer, an indexer for the explorer, and a NEAR wallet.
```

I don't believe we spin up anything else. Let's just be super specific about what it does.